docs/tutorial/part-three/index.md

```diff
@@ -1,57 +1,57 @@
 ---
-title: Creating Nested Layout Components
+शीर्षक: नेस्टेड लेआउट कम्पोनेंट्स बनाना
 typora-copy-images-to: ./
 disableTableOfContents: true
 ---
 
-Welcome to part three!
+भाग तीन में आपका स्वागत है!
 
-## What's in this tutorial?
+## इस ट्यूटोरियल में क्या है?
 
-In this part, you'll learn about Gatsby plugins and creating "layout" components.
+इस भाग में, आप गैट्सबी प्लगइन्स और "लेआउट" घटक बनाने के बारे में जानेंगे।
 
-Gatsby plugins are JavaScript packages that help add functionality to a Gatsby site. Gatsby is designed to be extensible, which means plugins are able to extend and modify just about everything Gatsby does.
+Gatsby plugins जावास्क्रिप्ट पैकेज हैं जो एक Gatsby साइट में कार्यक्षमता जोड़ने में मदद करते हैं। Gatsby को एक्स्टेंसिबल के लिए डिज़ाइन किया गया है, जिसका अर्थ है कि प्लगइन्स Gatsby जो भी करता है उसके बारे में सिर्फ विस्तार और संशोधन करने में सक्षम है।
```

Comment:
`जिसका अर्थ है कि प्लगइन्स Gatsby जो भी करता है उसके बारे में सिर्फ विस्तार और संशोधन करने में सक्षम है` => `जिसका अर्थ है प्लगिन्स सब कुछ जो Gatsby  करता है उसको एक्सटेंड  और मॉडिफाई कर सकता है`

Reply:
`कर सकता है` => `कर सकते है`

I think this is the correct version of it.
Sorry for missing this in 1st review